cip-0009.md

```diff
@@ -36,27 +36,16 @@ Memo
 
         A 34-byte sender specified field stored in an asset send message
         
-Message ID Range
-
-        A message ID can be identified as any number within a range of values.  For example, because there are no message IDs between 33554432 and 50331647 (equivalent to 02000000:02ffffff in hex format), we can define the message ID as any number falling within that range.  
-        
-Message ID Value Range
-
-        Data within a message ID range is referred to as the message type value range.  It can be used for any  purpose in the Counterparty message data.
-        
 
 # Overview
 
-An Enhanced Asset Send adds the following three features to a traditional Asset Send transaction...
+An Enhanced Asset Send adds the following two features to a traditional Asset Send transaction...
 
 
 ### Encoding Receiving Address in Message Data
 
 A standard Bitcoin/Counterparty Base58 receiving address can be hex encoded and stored as an element within a send message at a size of 21 bytes after removing the checksum. 
```

Comment:
Can we change `hex` to `binary` to be a little more specific?

A standard Bitcoin/Counterparty public key hash can be binary encoded and stored as an element within a send message at a size of 21 bytes after removing the checksum.